arkane/encorr/data.py

```diff
@@ -324,6 +326,20 @@
         'O': -74.97847534 + SOC['O'], 'S': -397.6571654 + SOC['S']
     },
 
+    # Calculated atomic energies fitted orca 4.2.1  dlpno-ccsd(t)/def2-tzvp NormalPNO
+    # fitted using Colin's BAC algorithm and SOC are included in the correction
+    # AEs are fitted to neutral molecules with RMSE and MAE 7.99 and 5.96 kJ/mol respectively.
+    'dlpno-ccsd(t)/def2-tzvp': {
+        'H': -0.49641082, 'C': -37.77274125, 'N': -54.50388932, 'O': -74.96760414,
+        'F': -99.62044819, 'S': -397.63480236, 'Cl': -459.65784960
+    },
+    # wb97xd/def2tzvp conducted using G16. 
+    # fitted using Colin's BAC algorithm and SOC are included in the correction
+    # AEs are fitted to neutral molecules with RMSE and MAE 9.16 and 6.66 kJ/mol respectively.
+    'wb97xd/def2tzvp': {
+        'H': -0.50224721, 'C': -37.84235323, 'N': -54.58757237, 'O': -75.06982154,
+        'F': -99.74180094, 'S': -398.10765642, 'Cl': -460.14650064
+    },
```

Comment:
Instead of manually adding in the SOC terms, write the energy and explicitly write in the SOC correction. For example, the correction for oxygen for dlpno-ccsd(t)/def2-tzvp would be `'O': -74.96724914 + SOC['O']` (which will get evaluated to -74.96760414). Do this for all atoms, even if the SOC is zero. This is a very minor thing, but it is better just in case we ever need to update the SOC values, and it makes it very clear that the correct SOC values were used.

Reply:
Figured I would chime in here. So these were fitted and not just calculated?